src/assets/scss/variables/_fonts.variables.module.scss

```diff
@@ -17,71 +17,71 @@ $font-weight-black: 900;
 
 // headlines
 // h1
-$font-size-h1: 60;
+$font-size-h1: 60px;
```

Comment:
Just curious why did you add the units to it, take into account that if you add the units, we cant use the "responsive" fonts

Reply:
I fixed the Typography Stories on my PR if you want to take a look the end result with one or another config